### PR TITLE
Reflected that HTTP was updated again in 2022 with RFC 9110

### LINKED ISSUE #34374: HTTP Spec was updated in 2022: RFC 9110+
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Evolution_of_HTTP

### What specific section or headline is this issue about?

More than 15 years of extensions

### What information was incorrect, unhelpful, or incomplete?

Text reads as if [RFC 7230](https://datatracker.ietf.org/doc/html/rfc7230) is the current/final HTTP spec.

### What did you expect to see?

Reflect that HTTP was updated again in 2022 with [RFC 9110](https://datatracker.ietf.org/doc/html/rfc9110)

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/http/basics_of_http/evolution_of_http`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Evolution_of_HTTP
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/http/basics_of_http/evolution_of_http/index.md
* Last commit: https://github.com/mdn/content/commit/fea9eb7420e6d70772144854242c30e421898415
* Document last modified: 2024-05-13T12:10:43.000Z

</details>

### PR DESCRIPTION
**Pull request by Ashish-CodeJourney**
### Description

Updated the documentation to reflect that HTTP was updated in 2022 with RFC 9110.

### Motivation

The previous documentation did not include the most recent update to HTTP, which occurred in 2022 with RFC 9110. This update is important for providing readers with the most current information regarding the HTTP protocol.

### Additional details

- [RFC 9110 - HTTP Semantics](https://www.rfc-editor.org/rfc/rfc9110.html)
- [HTTP/1.1 Overview](https://httpwg.org/specs/rfc9110.html)

### Related issues and pull requests

Fixes #34374

### CODE REVIEW

File: files/en-us/web/http/basics_of_http/evolution_of_http/index.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ HTTP/1.1 was first published as {{rfc(2068)}} in January 1997.
 
 ## More than 15 years of extensions
 
-The extensibility of HTTP made it easy to create new headers and methods. Even though the HTTP/1.1 protocol was refined over two revisions, {{RFC("2616")}} published in June 1999 and {{RFC("7230")}}-{{RFC("7235")}} published in June 2014 before the release of HTTP/2, it was extremely stable for more than 15 years.
+The extensibility of HTTP made it easy to create new headers and methods. Even though the HTTP/1.1 protocol was refined over two revisions, {{RFC("2616")}} published in June 1999 and {{RFC("7230")}}-{{RFC("7235")}} published in June 2014 before the release of HTTP/2, it was extremely stable for more than 15 years. HTTP/1.1 was updated again in 2022 with {{RFC("9110")}}.
```
Comment (JannisBush) [inline files/en-us/web/http/basics_of_http/evolution_of_http/index.md:145]: Not only HTTP/1.1 was updated, but all of HTTP which is now split into semantics (9110), caching (9111) applying to all HTTP versions and HTTP/1.1 (9112), HTTP/2 (9113) and HTTP/3 (9114).

In addition, the spec finally achieved the status of internet standard (STD 97) whereas before it was always a proposed/draft standard.

Comment (Ashish-CodeJourney) [reply files/en-us/web/http/basics_of_http/evolution_of_http/index.md]: Incorporated ! Please review

Comment (JannisBush) [reply files/en-us/web/http/basics_of_http/evolution_of_http/index.md]: They year (2022) got lost in the update.

Comment (Ashish-CodeJourney) [reply files/en-us/web/http/basics_of_http/evolution_of_http/index.md]: done, thanks for review :)

File: files/en-us/web/http/basics_of_http/evolution_of_http/index.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ HTTP/1.1 was first published as {{rfc(2068)}} in January 1997.
 
 ## More than 15 years of extensions
```
Comment (bsmth) [inline files/en-us/web/http/basics_of_http/evolution_of_http/index.md:143]: The only last thing to look at is maybe changing the section title here:

```suggestion
## More than two decades of development
```

**Review by bsmth — APPROVED** [review]
Thanks for the improvements, going to merge shortly!

**Comment by pransh15** [comment]
Well done, @Ashish-CodeJourney! 👏

**Pull request merged**